packages/outputarea/src/widget.ts

```diff
@@ -535,9 +535,17 @@ export namespace OutputArea {
     metadata?: JSONObject
   ): Promise<KernelMessage.IExecuteReplyMsg> {
     // Override the default for `stop_on_error`.
+    let stopOnError = true;
+    if (
+      metadata &&
+      metadata.tags &&
+      (metadata.tags as string[]).indexOf('raises-exception') !== -1
```

Comment:
`Array.isArray(metadata.tags)` for robustness?

Reply:
Good call, thanks!  That also removed the need for the type-cast.